nuxeo-java-client-test/src/test/resources/test-portal-sso-login-contrib.xml

```diff
@@ -0,0 +1,62 @@
+<component name="org.nuxeo.ecm.platform.authenticator.portal.sso.config">
+
+    <require>org.nuxeo.ecm.platform.ui.web.auth.WebEngineConfig</require>
+    <require>org.nuxeo.ecm.platform.login.Portal</require>
```

Comment:
Don't require `org.nuxeo.ecm.platform.login.Portal` since the `PORTAL_AUTH` plugin shouldn't be overridden.